lib/ecto/repo/model.ex

```diff
@@ -234,6 +254,7 @@ defmodule Ecto.Repo.Model do
   end
 
   defp with_transactions_if_callbacks(repo, adapter, model, opts, callbacks, fun) do
+    # TODO how should that work with embeds
```

Comment:
If `embeds != []`, we always wrap in a transaction (if the adapter supports so).